webapp/client/public/css/base.css

```diff
@@ -515,6 +515,17 @@ img.icon {
   color: white;
 }
 
+.alert-retirement {
+  background-color: #CCDDE4;
```

Comment:
i added a new colour in the variables.css `--blue-100: #CCDDE4;`

Reply:
done!